x-pack/plugins/session_view/public/components/ProcessTree/index.tsx

```diff
@@ -105,7 +107,7 @@ export const ProcessTree = ({
         const isVisible = eTop >= cTop && eBottom <= cBottom;
 
         if (!isVisible) {
-          processEl.scrollIntoView();
+          // processEl.scrollIntoView();
```

Comment:
should we keep this?

Reply:
yup, just disabled for demo, needs tweaking.